### PR TITLE
Pull RC changes back into develop

### PR DESCRIPTION
**Pull request by jklenzing**

### CODE REVIEW

**Comment by jklenzing** [comment]
pinging @aburrell

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -47,9 +48,20 @@ jobs:
       continue-on-error: true
       run: pip install aacgmv2 --no-binary==aacgmv2
 
-    - name: Install apexpy
-      # Skip apexpy for nep29 tests
-      if: ${{ matrix.numpy_ver == 'latest'}}
+    - name: Install apexpy for MacOS X
+      # Only test apexpy for nep29 tests
```
Comment (aburrell) [inline .github/workflows/main.yml]:
```suggestion
      # Only test apexpy for NEP29 tests
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -47,9 +48,20 @@ jobs:
...
+      # Optional package, continue with tests if install fails
+      continue-on-error: true
+      run: |
+          brew reinstall gcc
+          /opt/homebrew/bin/gfortran --version
+          /opt/homebrew/bin/gcc-14 --version
+          CC=/opt/homebrew/bin/gcc-14 FC=/opt/homebrew/bin/gfortran pip install apexpy --no-binary==apexpy
+
+    - name: Install apexpy for Unix/Windows
+      # Only test apexpy for nep29 tests
```
Comment (aburrell) [inline .github/workflows/main.yml]:
```suggestion
      # Only test apexpy for NEP29 tests
```

**Review by aburrell — COMMENTED** [review]
Small formatting improvement.

**Review by aburrell — APPROVED** [review]

**Pull request merged**